packages/analytics-js-integrations/src/integrations/VWO/browser.js

```diff
@@ -42,6 +42,17 @@ class VWO {
     } else {
       logger.debug('===[VWO]loadIntegration flag is disabled===');
     }
+
+    window.VWO = window.VWO || [];
+    window.VWO.event = window.VWO.event || function () {
+        window.VWO.push(['event'].concat([].slice.call(arguments)));
```

Comment:
ESLint: Use the rest parameters instead of 'arguments'.(prefer-rest-params)

Reply:
Addressed